resource-agents.spec.in

```diff
@@ -100,12 +100,13 @@ BuildRequires:  libxslt docbook_4 docbook-xsl-stylesheets
 Requires: /usr/bin/gawk
 Requires: /usr/bin/ps
 Requires: /usr/sbin/fuser /usr/bin/mount
+Requires: hostname
 %else
 Requires: /bin/gawk
 Requires: /bin/ps
 Requires: /sbin/fuser /bin/mount
+Requires: /bin/hostname
 %endif
-Requires: bash grep hostname sed
```

Comment:
You should keep the other dependencies here.

Reply:
I corrected the parts you pointed out. Please check it again.

Reply:
It's still missing some of the requirements.

Can you just undo the spec-file changes, as I dont think it's a good idea to start changing it until we have improved CI to ensure that it's working correctly on RHEL7/CentOS/*Suse, and the spec-file for e.g. RHEL6 is available in the SRPM.

Reply:
> You should keep the other dependencies here.

I'm sorry, I misunderstood.
You mean https://github.com/ClusterLabs/resource-agents/pull/1308/commits/c0845f4af0fa288e8f73e4e9ce07c695b336ee25, right?

Reply:
Correct.